Remove Application Registry and Connector Service from the Kyma documentation (#13984)
kyma-project/kyma: docs/05-technical-reference/00-custom-resources/ac-01-application.md

| **spec.services** | No | Contains all services that the Application provides. |
| **spec.services.id** | Yes | Identifies the service that the Application provides. |
| **spec.services.identifier** | No | Represents an additional identifier unique in the Application scope. Allows the external system to provide its own identifier. |
| **spec.services.name** | No | Represents a unique name of the service. Used for proxying in Application Gateway. |

> | **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.providerDisplayName** | Yes | Specifies a human-readable name of the Application service provider. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

Author: I think this is mostly preferential. I prefer to link the smallest meaningful bit, and I feel like `standalone` is just part of it. I'm going to leave it as is :)

Author: I replied before I noticed that you actually removed duplication here. Thanks!

> | **spec.services.tags** | No | Specifies additional tags used for better documentation of the available APIs. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.tags** | No | Specifies additional tags used for better documentation of the available APIs. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.labels** | No | Specifies additional labels for the service offered by the Application. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.labels** | No | Specifies additional labels for the service offered by the Application. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.entries** | Yes | Contains the information about the APIs and events that the service offered by the Application provides. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries** | Yes | Contains the information about the APIs and events that the service offered by the Application provides. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.entries.type** | Yes | Specifies the entry type: API or event. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries.type** | Yes | Specifies the entry type: API or event. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. The internal address is resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

Reviewer: Also: is `resolvable` a good word here? 🤔

Author: Yep. And address that can be resolved only within the cluster = is resolvable only within the cluster. 
I'll apply the `mode` bit, but I'll keep the original description, because it's the second part of the description of the parameter, not additional info.

| **spec.services.entries.accessLabel** | No | Specifies the label used in Istio rules in Application Connector. This field is required for the API entry type. |

> | **spec.services.entries.targetUrl** |  No | Specifies the URL of a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries.targetUrl** |  No | Specifies the URL of a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.entries.oauthUrl** | No | Specifies the URL used to authorize with a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries.oauthUrl** | No | Specifies the URL used to authorize with a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

> | **spec.services.entries.credentialsSecretName** | No | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md) mode, you have to provide it yourself. |

Reviewer: ```suggestion
| **spec.services.entries.credentialsSecretName** | No | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
```

## Related resources and components

These components use this CR:

| Component   |  Description |
|-----------|-------------|
| Application Gateway | Reads the API metadata in order to connect to the external system. | 

> | Runtime Agent (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR, synchronises the metadata stored in Compass with the state in the cluster stored in the Application CR. |

Reviewer: ```suggestion
| Runtime Agent (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR and synchronizes the metadata stored in Compass with the state in the cluster stored in the Application CR. |
```

> | Runtime Agent (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Saves the metadata of the connected external system in the Application CR, synchronises the metadata stored in Compass with the state in the cluster stored in the Application CR. |

Reviewer 2: I think Central Application Connectivity validator should be also included here

Author: What does it do with the CR? If I include it, I also need to add a description of how CACV uses it/what it uses it for.

Author: I received input via PM:

1) 
> The Application Connectivity Validator validates requests from external system with the `X-Forwarded-Client-Cert` header that contains `Subject` with the following fields corresponding to the Application custom resource:
> - **CommonName** is the name of the Application custom resource.
> - (Optional) **Organization** is the tenant.

2)

> The Application Connectivity Validator validates requests/events from external system against respective Application custom resource

3)

> Application Connectivity Validator makes sure that the events originating from registered External Systems end up in the respective workload, blocking any events that are not sent with the proper mTLS certificate.

I decided to go with:
> | Application Connectivity Validator (in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md)) | Validates requests and events from the external system against the respective Application CR. |
